### PR TITLE
[11.0][ADD] hr_course

### PR DESCRIPTION
**Pull request by jarroyomorales**
This module will allow to manage employee's training courses and all its validation process.

### CODE REVIEW

File: hr_course/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+{
+    'name': 'Hr Course',
+    'summary': """
+        This module allows your to manage employee's training courses and
+         all its validation process""",
+    'version': '11.0.1.0.0',
+    'license': 'AGPL-3',
+    'author': 'Creu Blanca,Odoo Community Association (OCA)',
+    'website': 'www.creublanca.es',
```
Comment (JordiBForgeFlow) [inline hr_course/__manifest__.py]: Replace with https://github.com/OCA/hr

**Review by etobella — APPROVED** [review]
LGTM :+1:

File: hr_course/views/hr_course_category_views.xml [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+            <field name="view_mode">tree,form</field>
+        </record>
+
+        <menuitem
+                id="menu_view_course_category_form"
+                action="open_view_course_category_form"
+                parent="hr.menu_human_resources_configuration"
+                sequence="10"/>
+    </data>
+</odoo>
```
Comment (HviorForgeFlow) [inline hr_course/views/hr_course_category_views.xml]: NIt: Add end blank line

**Review by HviorForgeFlow — APPROVED** [review]
code review :+1:

File: hr_course/models/hr_course.py [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+        'hr.course.attendee',
+        inverse_name='course_id',
+        readonly=True,
+        states={'in_validation': [('readonly', False)]},
+    )
+
+    @api.constrains('start_date', 'end_date')
+    def _check_start_end_dates(self):
+        self.ensure_one()
+        if self.start_date and self.end_date:
```
Comment (nikul-serpentcs) [inline hr_course/models/hr_course.py]: @jarroyomorales  IMO `if self.start_date and self.end_date and self.start_date > self.end_date:`

File: hr_course/security/course_security.xml [hunk 4]
```diff
@@ -0,0 +1,33 @@
+<?xml version="1.0"?>
+<odoo>
```
Comment (nikul-serpentcs) [inline hr_course/security/course_security.xml]: `<odoo noupdate="1">`

File: hr_course/views/hr_course_category_views.xml [hunk 5]
```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <data>
```
Comment (nikul-serpentcs) [inline hr_course/views/hr_course_category_views.xml]: no need `<data>`

File: hr_course/tests/test_hr_course.py [hunk 6]
```diff
@@ -0,0 +1,68 @@
+import odoo.tests.common as common
```
Comment (nikul-serpentcs) [inline hr_course/tests/test_hr_course.py:4]: Add Copyright

**Review by nikul-serpentcs — COMMENTED** [review]
Minor Change

**Comment by jarroyomorales** [comment]
@nikul-serpentcs Comments attendend!

**Comment by jarroyomorales** [comment]
@pedrobaeza @alexey-pelykh @jbeficent Can this be merged? :smiley:

**Review by JordiBForgeFlow — APPROVED** [review]

**Comment by HviorForgeFlow** [comment]
@jarroyomorales https://travis-ci.org/OCA/hr/jobs/598543246 check lint errors

**Comment by jarroyomorales** [comment]
@hveficent @jbeficent Yes sorry! Already fixed!

**Comment by jarroyomorales** [comment]
Green already :green_apple: , can we re run the bot?

**Pull request merged**